home-application/pom.xml

```diff
@@ -13,6 +13,11 @@
 
     <artifactId>home-application</artifactId>
     <dependencies>
+        <dependency>
+            <groupId>io.github.perplexhub</groupId>
+            <artifactId>rsql-jpa-spring-boot-starter</artifactId>
+            <version>5.0.6</version>
```

Comment:
Move versions to property section please

Reply:
Agree